src/sos/tasks.py

```diff
@@ -1685,26 +1685,26 @@ def row(th=None, td=None):
     elif verbosity == 0:
         print("\n".join(obtained_status))
     elif verbosity == 1:
-        for s, (t, d) in zip(obtained_status, all_tasks):
+        for s, (t, dr) in zip(obtained_status, all_tasks):
```

Comment:
what is the reason for this change?

Reply:
There was a warning that stated that d (line 1688) was an unused variable. I concluded that it was a typo. Should I revert it back to d?

Reply:
I just checked the latest commit on my GitHub page. If you navigate to https://github.com/jenningsje/Vat_sos/blob/consider-using-f-string/src/sos/tasks.py, you will find that dr has been replaced with underscore for lines 1688, 1692, 1699, 1707.